tests/python_tests/test_kv_cache_eviction.py

```diff
@@ -236,7 +238,7 @@ def test_optimized_generation_longbench(test_struct):
     generation_config.num_return_sequences = 1
     generation_config.max_new_tokens = max_new_tokens
 
-    data = datasets.load_dataset('THUDM/LongBench', subset, split='test[:32]', trust_remote_code=True)
+    data = datasets.load_dataset('THUDM/LongBench', subset, split='test[:32]', trust_remote_code=True, cache_dir=os.path.join(os.environ.get('OV_CACHE', tempfile.TemporaryDirectory().name), 'datasets'))
```

Comment:
https://huggingface.co/docs/datasets/cache#cache-directory
Datasets cache is enabled by default and points to HF_HOME. It set in workflows envs.
Anyway shouldn't point to OV_CACHE